### ISSUE TITLE
 [RTL] The arrow icons in the Settings menu are reversed

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#6540]
See screenshot:
https://drive.google.com/file/d/1M_Q_ZpKgDVT6X8MCSCPZIkvOm-8OoF45/view

The arrows should be facing left.

via : https://bugzilla.mozilla.org/show_bug.cgi?id=1634929

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2395)

**Comment by SimonBasca** [comment]
Verifying as fix on main 228b77b9a89f133417cb748fcf720fb01e6b6f1a. The arrows are facing the correct way in RTL
![image](https://user-images.githubusercontent.com/32130829/152153047-1063f295-b4e3-45b9-b776-b68711eae50e.png)

**Comment by SimonBasca** [comment]
Verified fix on v98.0 (8023).
